### PR TITLE
Add français locales

### PR DESCRIPTION
**Pull request by Cryptobunkers**

### CODE REVIEW

File: fr/common.json [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+		"notification_level_going_expire_detail": "Recommencez à <linkStaking>bloquer vos $PARAS</linkStaking> pour les conserver à {{currLevel}}",
+		"notification_raffle_type_drop_bronze_and_active": "<semibold>Votre niveau est tombé à Bronze !</semibold>",
+		"notification_raffle_type_drop_bronze_and_active_detail": "Recommencez à <linkStaking>bloquer vos $PARAS</linkStaking> pour rester inscrit dans le tirage au sort !",
+		"notification_raffle_type_drop_not_bronze_and_active": "<semibold>Votre niveau d'adhésion a diminué !</semibold>",
+		"notification_raffle_type_drop_not_bronze_and_active_detail": "Vous serez automatiquement inscrit au tirage au sort {{currRaffleType}}",
+		"notification_raffle_type_drop_bronze_and_deactive": "<semibold>Votre niveau est tombé à Bronze !</semibold>",
+		"notification_raffle_type_drop_bronze_and_deactive_detail": "Vous êtes actuellement membre {{currRaffleType}}. Recommencez à <Link>bloquer vos $PARAS</Link> pour les conserver à {{prevRaffleType}}"",
+		"notification_raffle_type_drop_not_bronze_and_deactive": "<semibold>Votre niveau est tombé à Bronze !</semibold>",
```
Comment (ahnafalfariza) [inline fr/common.json]: There is two double quote here causing json error

**Review by ahnafalfariza — COMMENTED** [review]
Hi @Cryptobunkers, thanks for the contribution!
Anyway, there is a format issue.

**Comment by Cryptobunkers** [comment]
Hi Ahnaf Alfariza,

The error is corrected

Le lun. 30 janv. 2023 à 06:28, Ahnaf Alfariza ***@***.***> a
écrit :

**Comment by ahnafalfariza** [comment]
Thanks!
Here is the deployed version of paras website in french (https://staging-new.paras.id/fr)
Please update if there are translations that are wrong or not quite a right fit

In the meantime, we will check also on our side
cc: @ekkirinaldi

**Comment by Cryptobunkers** [comment]
I have updated some translations to fit better. Can you upload files I
updated today ?

Le lun. 30 janv. 2023 à 08:51, Ahnaf Alfariza ***@***.***> a
écrit :

**Comment by ahnafalfariza** [comment]
Hi @Cryptobunkers, could you please fix this error, mainly about the missing coma `,` at the end of the line

image

**Comment by Cryptobunkers** [comment]
Hi, it's done.
I have understood why the comma were missing 👌


Le mer. 8 févr. 2023 à 09:31, Ahnaf Alfariza ***@***.***> a
écrit :

**Pull request merged**